src/Gui/MruList.cs

```diff
@@ -31,16 +31,22 @@ namespace Reko.Gui
 	/// </summary>
 	public class MruList
 	{
-		private int itemsMax;
-		private List<string> items;
+        public struct fileOutputDir
+        {
+            public string filename;
+            public string outputDir;
+        };
```

Comment:
I think, changing `Mru` format is not good idea. It's just the list of last opened files and should contain file names only

Reply:
The MRU (most recently used file) list should just contain file names. The output directory name can be deduced for simple binaries, and for project files, they should contain a property called `OutputDirectoryName` which would allow users to modify the location of the directory if they want.